src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
Probably would make more sense to put it in project root and configure point to it: `"setupTestFrameworkScriptFile": "./setupTest.js"`.
After installing *jest-styled-components* `import 'jest-styled-components'` could be added there as well.